reducer/src/main/java/com/graphicsfuzz/reducer/ReductionDriver.java

```diff
@@ -119,7 +119,8 @@ public ReductionDriver(ReducerContext context,
         IReductionOpportunityFinder.liveFragColorWriteFinder(),
         IReductionOpportunityFinder.functionFinder(),
         IReductionOpportunityFinder.variableDeclFinder(),
-        IReductionOpportunityFinder.globalVariablesDeclarationFinder())) {
+        IReductionOpportunityFinder.globalVariablesDeclarationFinder(),
```

Comment:
I regard this as a cleanup pass rather than a core pass; can you move it up so that it's only added to cleanup passes?

Reply:
Fixed